.github/workflows/maven-build.yaml

```diff
@@ -28,4 +28,5 @@ jobs:
           CLIENT_CREDENTIALS_URL: ${{ secrets.CLIENT_CREDENTIALS_URL }}
         run: |
           export JAVA_TOOL_OPTIONS="--add-opens=java.base/java.nio=ALL-UNNAMED"
+          mkdir -p ~/.rai
```

Comment:
How only making this dir will allow to cache the token? Do we already have code in the SDK caching token in this dir?

Reply:
yep that's the case